pkg/store/bucket.go

```diff
@@ -221,6 +224,18 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Help: "Compressed size of postings stored into cache.",
 	})
 
+	m.seriesLookupDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+		Name:    "thanos_bucket_store_index_series_lookup_duration_seconds",
+		Help:    "Time it takes to lookup series from a bucket to respond a query. It also includes the cache fetch and store operations.",
+		Buckets: []float64{0.001, 0.01, 0.1, 0.3, 0.6, 1, 3, 6, 9, 20, 30, 60, 90, 120},
+	})
+
+	m.postingsLookupDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
```

Comment:
Can we have just label? It will much easier to aggregate, plus we already do that for other statistics, WDYT?

I think also bucket can be adjusted for timeout. BTW What lookup means is quite vague by this help :thinking:  Maybe we can be explicit more.

Reply:
@bwplotka The first version was using a single histogram with labels https://github.com/thanos-io/thanos/pull/2725/commits/8a46f33ac88716c559ac1c0eeefe29143abc04e6. Then I decided to change it, just to be consistent with what we already have in here.

> BTW What lookup means is quite vague by this help 🤔 Maybe we can be explicit more.

I'm gonna change the name as you suggested and try to clarify the description.

> I think also bucket can be adjusted for timeout. 

I've put the boundary at the query timeout. Which timeout value is used for this?

Reply:
@bwplotka friendly ping.

Reply:
ok for now :+1: